selfswab/tasks.py

```diff
@@ -33,7 +34,10 @@ def poll_meditech_api_for_results():
             )
             response = requests.post(
                 url=settings.MEDITECH_URL,
-                headers={"Content-Type": "application/json"},
+                headers={
+                    "Content-Type": "application/json",
+                    "Authorization": f"Basic {settings.MEDITECH_AUTH}",
```

Comment:
Would it not be better to use username and password, and have `requests` do the base64 encoding for us? Since it does have that feature.

Reply:
Probably better yes